components/Slider/src/MDCSlider.m

```diff
@@ -544,8 +553,10 @@ - (void)thumbTrackValueChanged:(__unused MDCThumbTrack *)thumbTrack {
   [self sendActionsForControlEvents:UIControlEventValueChanged];
   UIAccessibilityPostNotification(UIAccessibilityAnnouncementNotification, self.accessibilityValue);
   if (@available(iOS 10.0, *)) {
-    if (self.hapticsEnabled && (_thumbTrack.value == _thumbTrack.minimumValue ||
-                                _thumbTrack.value == _thumbTrack.maximumValue)) {
+    if (self.fullHapticsEnabled) {
```

Comment:
what happens if hapticsEnabled is set to NO, do we still want haptics to occur if fullhapticsenabled is set to YES?